src/shared/sources/th/index.js

```diff
@@ -0,0 +1,60 @@
+const assert = require('assert')
+const maintainers = require('../_lib/maintainers.js')
+
+const UNASSIGNED = '(unassigned)'
+
+module.exports = {
+  aggregate: 'state',
+  country: 'iso1:TH',
+  priority: 1,
+  friendly: {
+    name: 'Thailand Ministry of Public Health',
+    url: 'https://ddc.moph.go.th/viralpneumonia/eng/index.php'
+  },
+  maintainers: [ maintainers.camjc ],
+  scrapers: [
+    {
+      startDate: '2020-05-05',
+      crawl: [
+        {
+          name: 'states',
+          type: 'json',
+          url: 'https://ddcportal.ddc.moph.go.th/arcgis/rest/services/iT_Neillgis/thai_prov_region/FeatureServer/0/query?where=0%3D0&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=*&returnGeometry=false&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pjson'
```

Comment:
Wondering if we should have a common constant for 'give me all the data please ArcGIS' query string.